### PR TITLE
Qa tsweb 1135

### PR DESCRIPTION
**Pull request by KotliarovaK**

### CODE REVIEW

File: ui_tests_playwright/automation/identifiers/MainSite/pages/services/DigitalTransformation.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import GeneralContainersMainSite from '../../GeneralContainersMainSite';
+
+export default class AiDevelopment extends GeneralContainersMainSite {
+    static OurAchievementsInITTransformation = 'Container-OurAchievementsInITTransformationServices';
+    static DigitalBusinessTransformation = 'Container-DigitalBusinessTransformationServices';
+    static IndustriesWeServe = 'Container-IndustriesWeServe';
+    static SuccessStories = 'Container-SuccessStories';
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/identifiers/MainSite/pages/services/DigitalTransformation.ts:7]: Seems like these (not only) data-ids already exist. Please run unit tests 
npm run unit-tests

File: ui_tests_playwright/automation/test/ui/MainSite/services/DigitalTransformation/DigitalTransformationContent.test.ts [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+import UrlPath from '../../../../../providers/UrlPath';
+import DigitalTransformation from '../../../../../identifiers/MainSite/pages/services/DigitalTransformation';
+import Container from '../../../../../identifiers/Container';
+import MainSiteImages from '../../../../../identifiers/MainSite/MainSiteImages';
+
+test.beforeEach(async () => {
+	await baseDriverSteps.createsNewBrowserAndGoToUrl(UrlProvider.urlBuilder(UrlPath.DigitalTransform));
+});
+
+test('Check the Info container from the "Digital Transformation Services" page @Regression @DigitalTransformation @TSWEB-1135', async () => {
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/services/DigitalTransformation/DigitalTransformationContent.test.ts:15]: *only comment
mb just "Digital Transformation" page ?

File: ui_tests_playwright/automation/test/ui/MainSite/services/DigitalTransformation/DigitalTransformationContent.test.ts [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+	const containers = [
+		driver.getByTestId(DigitalTransformation.OurAchievementsInITTransformation),
+		driver.getByTestId(DigitalTransformation.DigitalBusinessTransformation),
+		driver.getByTestId(DigitalTransformation.IndustriesWeServe),
+		driver.getByTestId(DigitalTransformation.SuccessStories),
+		driver.getByTestId(DigitalTransformation.TechnologiesWeUse),
+		driver.getByTestId(DigitalTransformation.DigitalTransformationStrategy),
+		driver.getByTestId(DigitalTransformation.HowTechstackInfluence),
+		driver.getByTestId(DigitalTransformation.DigitalTransformationProductMap),
+        driver.getByTestId(DigitalTransformation.TechnologyTransformationWorkflow),
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/services/DigitalTransformation/DigitalTransformationContent.test.ts:32]: smt with format (or only remote?)

File: ui_tests_playwright/automation/test/ui/MainSite/services/DigitalTransformation/DigitalTransformationContent.test.ts [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+
+test('Check section titles in "Our Achievements in IT Transformation Services" container from the "Digital Transformation Services" page @Regression @DigitalTransformation @TSWEB-1135', async () => {
+	const ourAchievementsContainer = driver.getByTestId(DigitalTransformation.OurAchievementsInITTransformation);
+	const allSectionTitles = ourAchievementsContainer.getByTestId(Container.SectionTitle);
+	const testDataSectionTitles = ['Our partner grew from 10 to 60 with our expertise', 'Mobile app & IoT device in 1 month', 'From MVP to unicorn in 7 years'];
+
+	await expect(allSectionTitles).toHaveText(testDataSectionTitles);
+});
+
+test('Check section numbers and titles in "Digital Business Transformation Services" container from the "Digital Transformation Services" page @Regression @DigitalTransformation @TSWEB-1135', async () => {
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/services/DigitalTransformation/DigitalTransformationContent.test.ts:64]: mb add checking CTA text?

test('Check section numbers and titles, **and CTA** in "Digital Busi....

await expect(Container.getById(MainSiteButtons.CTA)).toHaveText(*text*);

File: ui_tests_playwright/automation/test/ui/MainSite/services/DigitalTransformation/DigitalTransformationContent.test.ts [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+		' Operational processes',
+		' Business models',
+		' Process and workforce automation',
+	];
+
+	await expect(allSectionTitles).toHaveText(testData);
+});
+
+test('Check section numbers and section titles in "Industries We Serve" container from the "Digital Transformation Services" page @Regression @DigitalTransformation @TSWEB-1135', async () => {
+	const industriesWeServeContainer = driver.getByTestId(DigitalTransformation.IndustriesWeServe);
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/services/DigitalTransformation/DigitalTransformationContent.test.ts:81]: test('Check section numbers and titles ....

File: ui_tests_playwright/automation/test/ui/MainSite/services/DigitalTransformation/DigitalTransformationContent.test.ts [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+		'Sales and marketing',
+		'Human resources',
+		'AgroTech',
+		'Hospitality',
+	];
+
+	await expect(allSectionTitles).toHaveText(testData);
+});
+
+test('Check section titles, image and CTA button`s title in "Success Stories" container from the "Digital Transformation Services" page @Regression @DigitalTransformation @TSWEB-1135', async () => {
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/services/DigitalTransformation/DigitalTransformationContent.test.ts:111]: mb?
test('Check section titles, block title, image and CTA button in "Suc ...

and check for "IoT tracking and aggregation system scheme" block title (if needed)

**Review by Valerii-Kryvokhyzha — CHANGES_REQUESTED** [review]

**Comment by KotliarovaK** [comment]
no longer relevant

**Pull request closed without merging**